tests/e2e/specs/stories-editor/global-inserter.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * WordPress dependencies
+ */
+import { createNewPost, getAllBlocks } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { activateExperience, deactivateExperience, insertBlock } from '../../utils';
+
+const INSERTER_SELECTOR = '#amp-story-inserter';
+const CTA_BLOCK_NAME = 'Call to Action';
+
+/**
+ * Tests that a given block is present in the inserter.
+ *
+ * @param {string} block The name of the block, like 'Image'.
+ */
+const testBlockPresentInInserter = async ( block ) => {
```

Comment:
That could be a use case for a [custom matcher](https://jestjs.io/docs/en/expect#custom-matchers-api).